### PR TITLE
Update zh translation in dns-pod-service.md

### PR DESCRIPTION
**Pull request by wwgfhf**
Update zh translation in dns-pod-service.md

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ For more up-to-date specification, see
 ## 怎样获取 DNS 名字?
 
 在集群中定义的每个 Service（包括 DNS 服务器自身）都会被指派一个 DNS 名称。
-默认，一个客户端 Pod 的 DNS 搜索列表将包含该 Pod 自己的 Namespace 和集群默认域。
+默认情况下，一个客户端 Pod 的 DNS 搜索列表将包含该 Pod 自己的 Namespace 和集群默认域。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:50]: /LGTM

Comment (tanjunchen) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md:50]: @wwgfhf
Thanks

**Comment by tengqm** [comment]
Looks like we hit a transient netlify issue. Could you pls rebase this onto master and commit again?

**Comment by tengqm** [comment]
@wwgfhf pls rebase this one on to master?

**Comment by tengqm** [comment]
This one has been stalling for too long. Please feel free to reopen.
/close

**Pull request closed without merging**